reVX/least_cost_xmission/transmission_layer_creator_cli.py

```diff
@@ -0,0 +1,256 @@
+"""
+CLI to create wet (TODO and dry) costs, barriers, and friction layers. Final
```

Comment:
TODO?

Reply:
Planning to incorporate dry cost creation into the new CLI later.